lib/pages/character/inventory.dart

```diff
@@ -90,42 +90,56 @@ class InventoryPage extends StatelessWidget {
   Widget _buildBag(BuildContext context, Bags bag, int bagIndex) {
```

Comment:
I'm not a big fan of the changes to the Inventory page. Instead of giving the headers a different color (which is a bit too much imo), maybe add a separator in-between the header and the items?

Reply:
I've updated this with a Divider() widget instead of the coloured background, but only when there's something to show in the bag slot.